packages/graphql/tests/integration/connections/unions.int.test.ts

```diff
@@ -141,7 +141,7 @@ describe("Connections -> Unions", () => {
         }
     });
 
-    test("With `where` argument", async () => {
+    test("With `where` argument on node", async () => {
```

Comment:
We could probably do with a test validating that a node + relationship filter works as expected?

Reply:
Sure